packages/flutter_tools/test/ios/simulators_test.dart

```diff
@@ -94,6 +94,20 @@ void main() {
     });
   });
 
+  group('sdkMajorVersion', () {
+    test('can be parsed from iOS-11-3', () async {
```

Comment:
Is this a new format starting with Xcode 9.3? 

Consider adding a comment here indicating the conditions under which we expect to see this (Xcode >= 9.3, I assume), as well as one on the test below -- if it's a version-specific behaviour, the previous behaviour covers at least Xcode 7 to 9.2.

Reply:
I _think_ it is a beta version of the simulator. My xcode version was still 9.2 - I can verify this tonight

Reply:
update - I have the exact same versions (9.2) on both my work and personal macs, down to the builder number. 

Likewise, both simulators are version 10.0.  However, my personal computer has a "beta" ribbon across it and slightly different version names.

new: SimulatorApp-850 CoreSimulator-518.16
old: SimulatorApp-835.5 CoreSimulator-494.33

Reply:
added comment